src/Microsoft.ServiceFabric.Services.Remoting/V2/Runtime/ServiceRemotingMessageDispatcher.cs

```diff
@@ -67,13 +73,17 @@ public class ServiceRemotingMessageDispatcher : IServiceRemotingMessageHandler,
         /// <param name="serviceContext">Service context</param>
         /// <param name="serviceImplementation">Service implementation that implements interfaces of type <see cref="IService"/></param>
         /// <param name="serviceRemotingMessageBodyFactory">The factory that will be used by the dispatcher to create response message bodies.</param>
+        /// <param name="activitySourceName">Name of the ActivitySource to be used for tracing</param>
+        /// <param name="activitySourceVersion">Version of ActivitySource to be used for tracing</param>
         public ServiceRemotingMessageDispatcher(

```

Comment:
I believe you need to create overloaded constructor for ActorDispatcher as well.

Reply:
Yep, Added the support